composer.json

```diff
@@ -52,7 +53,7 @@
 		"symfony/yaml": "^4.4",
 		"twig/twig" : "^2.11",
 		"symfony/ldap": "^4.4",
-		"friendsofsymfony/user-bundle": "^2.1",
+		"paragonie/random_compat": "^1 || ^2",
```

Comment:
Why do we neeed this dep? Its a polyfill for php 5.x random_bytes() and random_int().
Our composer requires 7.2, this dep has to go and we can just use the native functions